pkg/engine/lifecycletest/import_test.go

```diff
@@ -1261,3 +1263,85 @@ func TestImportInputDiff(t *testing.T) {
 	assert.Equal(t, upInputs, snap.Resources[1].Inputs)
 	assert.Equal(t, readOutputs, snap.Resources[1].Outputs)
 }
+
+func TestImportDefaultProvider(t *testing.T) {
+	t.Parallel()
+
+	readInputs := resource.PropertyMap{
+		"foo": resource.NewStringProperty("bar"),
+	}
+	readOutputs := resource.PropertyMap{
+		"foo":  resource.NewStringProperty("bar"),
+		"frob": resource.NewNumberProperty(1),
+	}
+
+	pkgAVersion := semver.MustParse("1.0.0")
+	loaders := []*deploytest.ProviderLoader{
+		deploytest.NewProviderLoader("pkgA", pkgAVersion, func() (plugin.Provider, error) {
+			return &deploytest.Provider{
+				GetSchemaF: func(context.Context, plugin.GetSchemaRequest) (plugin.GetSchemaResponse, error) {
+					return plugin.GetSchemaResponse{Schema: []byte(importSchema)}, nil
+				},
+				DiffF: diffImportResource,
+				CreateF: func(_ context.Context, req plugin.CreateRequest) (plugin.CreateResponse, error) {
+					return plugin.CreateResponse{
+						ID:         "created-id",
+						Properties: req.Properties,
+						Status:     resource.StatusOK,
+					}, nil
+				},
+				ReadF: func(context.Context, plugin.ReadRequest) (plugin.ReadResponse, error) {
+					return plugin.ReadResponse{
+						ReadResult: plugin.ReadResult{
+							ID:      "actual-id",
+							Inputs:  readInputs,
+							Outputs: readOutputs,
+						},
+						Status: resource.StatusOK,
+					}, nil
+				},
+			}, nil
+		}),
+	}
+
+	programF := deploytest.NewLanguageRuntimeF(func(_ plugin.RunInfo, monitor *deploytest.ResourceMonitor) error {
+		return errors.New("unexpected program execution")
+	}, workspace.PackageDescriptor{
+		PluginSpec: workspace.PluginSpec{
+			Name:    "pkgA",
+			Kind:    apitype.ResourcePlugin,
+			Version: &pkgAVersion,
+		},
+	})
+	hostF := deploytest.NewPluginHostF(nil, nil, programF, loaders...)
+
+	p := &lt.TestPlan{
+		Options: lt.TestUpdateOptions{T: t, HostF: hostF, SkipDisplayTests: true},
```

Comment:
Any reason we want to skip the display tests for this one? I recall for instance we don't print out stuff about default providers; not sure if it's worth having this on so that we can track that behaviour going forward?